MLS.Agent.Tests/CommandLine/CommandLineParserTests.cs

```diff
@@ -59,9 +60,9 @@ public CommandLineParserTests(ITestOutputHelper output)
                     _installPackageSource = options.AddSource;
                     return Task.CompletedTask;
                 },
-                verify: (options, console, startupOptions) =>
+                verify: (directoryAccessor, console, startupOptions) =>
```

Comment:
How does this `directoryAccessor` relate to `startupOptions.Dir`? This seems like we have a concept being duplicated. Ideally we'd use `IDirectoryAccessor` (and the same instance) for both.

Reply:
Modified the command line parser to input a  directory and create a directory accessor which is used in the StartupOptions and the VerifyOptions